civcraftConfigs/alephconfig.yml

```diff
@@ -139,27 +139,39 @@ monster:
          enchant: KNOCKBACK
          level: 1
      equipment:
-      watch:
-       material: WATCH
+      BluntHotRod:
+       material: BLAZE_ROD
        enchants:
-        KB5:
+        KB8:
          enchant: KNOCKBACK
-         level: 5
+         level: 8
         S2:
          enchant: DAMAGE_ALL
          level: 2
       firechest451:
-       material: DIAMOND_CHESTPLATE
+       material: LEATHER_CHESTPLATE
        enchants:
         Prot451:
          enchant: PROTECTION_FIRE
          level: 451
-      fireBoots:
+      firelegs451:
+       material: LEATHER_LEGGINGS
+       enchants:
+        Prot451:
+         enchant: PROTECTION_FIRE
+         level: 451
+      fireBoots154:
        material: LEATHER_BOOTS
        enchants:
         FeatherFall4:
          enchant: PROTECTION_FALL
-         level: 4
+         level: 3
+        Prot451:
+         enchant: PROTECTION_FIRE
+         level: 451
+# Testing if the skelly can wear a black dyed glass head.
+     glasshelm:
+       material: GLASS;0
```

Comment:
DarkJesus is testing if this will make the skelly wear a black glass block on its head. I don't know if it'll work.